futurex_openedx_extensions/dashboard/statistics/certificates.py

```diff
@@ -41,3 +42,61 @@ def get_certificates_count(
     )).values('course_org').annotate(certificates_count=Count('id')).values_list('course_org', 'certificates_count'))
 
     return dict(result)
+
+
+def get_learning_hours_count(
+    fx_permission_info: dict, visible_courses_filter: bool | None = True, active_courses_filter: bool | None = None
+) -> int:
+    """
+    Get the count of learning hours in the given tenants. The count is grouped by course_effort. Certificates
+    for admins, staff, and superusers are also included.
+
+    :param fx_permission_info: Dictionary containing permission information
+    :type fx_permission_info: dict
+    :param visible_courses_filter: Value to filter courses on catalog visibility. None means no filter.
+    :type visible_courses_filter: bool | None
+    :param active_courses_filter: Value to filter courses on active status. None means no filter.
+    :type active_courses_filter: bool | None
+    :return: Count of certificates per organization
+    :rtype: Dict[str, int]
+    """
+
+    def parse_course_effort(effort: str) -> float:
+        """Parses course effort in HH:MM format and returns total hours as a float."""
+        try:
+            parts = effort.split(':')
+            hours = int(parts[0])
+            minutes = int(parts[1]) if len(parts) > 1 else 0
+
+            if minutes >= 60:
+                raise ValueError('Minutes cannot be 60 or more.')
+
+            total_hours = hours + minutes / 60
+            return round(total_hours, 1)
+        except (ValueError, IndexError):
+            return settings.FX_DEFAULT_COURSE_EFFORT
```

Comment:
we want to make it silent for the user, but noisy for us in the logs
```suggestion
            if total_hours < 0.5:
                raise ValueError('course effort value is too small')
            return round(total_hours, 1)
        except (ValueError, IndexError) as exc:
            log.exception('Invalid course-effort for course %s. Assuming default value (%s hours)', course_id, settings.FX_DEFAULT_COURSE_EFFORT)
            return settings.FX_DEFAULT_COURSE_EFFORT
```